code-challenges.test.js

```diff
@@ -26,42 +26,162 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // Pseudo code:
+    // Input: an object
+    // Output: a string that states whether the object is divisible by three, or is not divisible by three
+    // Process: create a function using % 3 === 0 operator to determine if a number is divisible by 3
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
+describe("threeDivider", () => {
+    it(" takes a object as an argument and decides if the number inside it is evenly divisible by three or not", () => {
+    const object1 = { number: 15 }
 // Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
+    const object2 = { number: 0 }
 // Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
+    const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
+        expect(threeDivider(object1.number)).toEqual("15 is divisible by three")
+        expect(threeDivider(object2.number)).toEqual("0 is divisible by three")
+        expect(threeDivider(object3.number)).toEqual("-7 is not divisible by three")
+        })
+})
+
+// test output: threeDivider
+    // ✕  takes a object as an argument and decides if the number inside it is evenly divisible by three or not (1 ms)
+
+    // ● threeDivider ›  takes a object as an argument and decides if the number inside it is evenly divisible by three or not
+  
+    //   ReferenceError: threeDivider is not defined
 
 // b) Create the function that makes the test pass.
 
+// Green:
+// const threeDivider = (object) => {
+//     if(object % 3 === 0)
+//     return `${object} is divisible by three`
+//     else 
+//     return `${object} is not divisible by three`
+// }
+
+// Refactor:
+const threeDivider = (object) => {
+    return object % 3 === 0 ? `${object} is divisible by three` : `${object} is not divisible by three`
+}
+
+// output: ./code-challenges.test.js
+//   threeDivider
+//   ✓  takes a object as an argument and decides if the number inside it is evenly divisible by three or not (3 ms)
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // Pseudo code:
+    // Input: an array of words
+    // Output: an array with all the words capitalized
+    // Process: create a function using the .map higher order function and .toUpperCase method to output all the given words capitalized
```

Comment:
Make sure to explain every part of your logic. In this case, you did not explain that you are going to concatenate the uppercased first letter with a substring of the original string starting at index 1 using `.substring`.